playbooks/roles/mysql_init/defaults/main.yml

```diff
@@ -44,6 +44,11 @@ mysql_database_users:
       pass: "{{ EDXAPP_MYSQL_PASSWORD | default(None) }}"
     }
   - {
+      db: "{{ EDXAPP_MYSQL_CSMH_DB_NAME | default(None) }}",
+      user: "{{ EDXAPP_MYSQL_CSMH_USER | default(None) }}",
+      pass: "{{ EDXAPP_MYSQL_CSMH_PASSWORD | default(None) }}"
```

Comment:
The `default(None)` is triggered when the variable is undefined. Without it, we'd get an error about the variable being undefined.